tests/test_opencode_sync.py

```
@@ -937,3 +528,326 @@ def test_check_orphans_clean_against_the_real_script() -> None:
+    pattern against the whole repo regardless of the actual diff, so it
+    fires on every CI run -- but `opencode` is never installed in CI
+    (design.md's Gap note: no automated harness runs OpenCode there). The
+    hook's own entry -- not verify()'s exit-non-zero-on-missing-binary
+    logic, which a developer invoking `bin/opencode-sync --verify` directly
+    still gets -- must detect the missing binary and skip with exit 0
+    rather than fail every CI run on an unrelated diff.
+    """
+    hook = _find_verify_hook()
+    argv = shlex.split(hook["entry"])
+
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)